t/plugin/key-auth.t

```diff
@@ -172,11 +172,34 @@ GET /hello
 
 
 === TEST 7: valid consumer
+--- config
+    location /add_more_consumer {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local username = ""
+            local key = ""
+            local code, body
+            for i = 1, 20 do
+                username = "user_" .. tostring(i)
+                key = "auth-" .. tostring(i)
+                code, body = t('/apisix/admin/consumers',
+                    ngx.HTTP_PUT,
+                    string.format('{"username":"%s","plugins":{"key-auth":{"key":"%s"}}}', username, key),
+                    string.format('{"node":{"value":{"username":"%s","plugins":{"key-auth":{"key":"%s"}}}},"action":"set"}', username, key)
+                    )
+            end
+
+            ngx.status = code
+            ngx.say(body)
+        }
+    }
 --- request
-GET /hello
+GET /add_more_consumer
+--- pipelined_requests eval
+["GET /add_more_consumer", "GET /hello"]
 --- more_headers
 apikey: auth-one
```

Comment:
`apikey: auth-one` -> `apikey: auth-13`, test the new code logic.

Reply:
![image](https://user-images.githubusercontent.com/11989090/68543553-713d4b00-03f3-11ea-8a89-0e6d5f9f34b0.png)

YES,UPDATED